Sources/Core/Shared/Operation.swift

```diff
@@ -61,6 +61,35 @@ public class Operation: NSOperation {
         }
     }
 
+    public class Errors {
+        public var all = Array<ErrorType>()
+        public var handled = Array<ErrorType>()
+
+        public var count: Int {
+            return all.count
+        }
+
+        public var isEmpty: Bool {
+            return all.isEmpty
+        }
+
+        public var first: ErrorType? {
+            return all.first
+        }
+
+        var didFinishWithErrors: Bool {
+            return count > 0 && count != handled.count
+        }
+
+        public func addErrors(errors: [ErrorType]) {
+            all += errors
+        }
+
+        public func handledErrors(errors: [ErrorType]) {
+            handled += errors
+        }
+    }
+
```

Comment:
Seems to me it would be quite useful to have an `Errors`.`unhandled` accessor too? Thoughts?